src/Missings.jl

```diff
@@ -514,4 +514,73 @@ julia> emptymissing(first)([1], 2)
 """
 emptymissing(f) = (x, args...; kwargs...) -> isempty(x) ? missing : f(x, args...; kwargs...)
 
-end # module
+# Only for internal use. Allows dispatch over anonymous functions.
+struct MissingSmallest{T}
+    lt::T
+end
+
+"""
+    missingsmallest(f)
+
+Creates a function of two arguments `x` and `y` that tests whether `x` is less
+than `y` such that `missing` is always less than the other argument. In other 
+words, returns a modified version of the partial order function `f` such that
+`missing` is the smallest possible value, and all other non-`missing` values are
+compared according to `f`.
+
+The traditional `isless` function modified to treat `missing` as the smallest
+value can be accessed by using the 2-argument `missingsmallest(x, y)` function,
+see [`missingsmallest`](@ref).
```

Comment:
```suggestion
The behavior of the standard `isless` function modified to treat `missing` as the smallest
value can be obtained by calling the 2-argument `missingsmallest(x, y)` function. This
is equivalent to `missingsmallest(isless)(x, y)`.
```
(The link points to the same docstring.)

Reply:
I'm not sure it's worth adding a reference, as both docstrings will be grouped in the manual and in the REPL.

Reply:
Yes - I think this cross reference is not needed in this case.